### PR TITLE
Style/ux changes implementation 165

### PR DESCRIPTION
**Pull request by annapakula**
- add icons in the left menu
- add more info on dashboard about employees
- change nomenclature of folders and forms
- change view of folders list - add description, drop down list with forms
- center warning modal

image
image

### CODE REVIEW

File: onboarding/src/Components/EmployeeProfile/ImplementationFormsToSend.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ function ImplementationFormsToSend(props) {
     return(
         <div className="card card-fluid">
             <div className="card-header">
+                <i class="bi bi-cloud-upload mr-2" style={{ fontSize: "18px" }}></i>
```
Comment (adam-sas-on) [inline onboarding/src/Components/EmployeeProfile/ImplementationFormsToSend.js:51]: `Warning: Invalid DOM property 'class'. Did you mean 'className'?`
`in i (created by ImplementationFormsToSend)`

File: onboarding/src/Components/hooks/PackagePage.js [hunk 2]
```diff
@@ -143,7 +100,7 @@ export function savePackageDetails(handleSuccess, packageId, title, description)
 /**
  * Add page into Pages for Package (todo: set owner as a logged HR manager?);
  */
-export function addPage(handleSuccess, title, packageId, order, owner){
+export function addPage(handleSuccess, title, order, packageId, owner){
```
Comment (adam-sas-on) [inline onboarding/src/Components/hooks/PackagePage.js:103]: This swappiness caused an error :hushed: - page was added to the wrong package :anguished:

File: onboarding/src/Components/hooks/Packages.js [hunk 3]
```diff
@@ -1,15 +1,13 @@
-import React, { useState, useEffect } from "react";
+import { useState, useEffect } from "react";
```
Comment (adam-sas-on) [inline onboarding/src/Components/hooks/Packages.js:1]: Removing this import causes e.g.:
`ESLint: React Hook "useState" is called in function "fetchPackages" which is neither a React function component or a custom React Hook function.(react-hooks/rules-of-hooks)`
`ESLint: React Hook "useEffect" is called in function "fetchPackages" which is neither a React function component or a custom React Hook function.(react-hooks/rules-of-hooks)`
This is underlined everywhere where `useEffect` or `useState` is called.

Comment (annapakula) [reply onboarding/src/Components/hooks/Packages.js:1]: I removed it, because JSX is not rendering in this file, so importing React here is not necessary.

**Pull request merged**